cfme/tests/cloud_infra_common/test_html5_vm_console_ssui.py

```diff
@@ -0,0 +1,147 @@
+"""Test for HTML5 Remote Consoles of VMware/RHEV/RHOSP Providers."""
+import pytest
+from wait_for import wait_for
```

Comment:
https://github.com/ManageIQ/integration_tests/blob/master/cfme/utils/wait.py

Please import wait_for through cfme.utils, so the logger context is passed.

Reply:
done